### ISSUE TITLE
"Error in L2/L3 Traffic Apply" response from TrafficItem's Generate() method. 

### ISSUE THREAD

**Issue by Brodski** [OpenIxia/IxNetwork#29]
I have some basic code and a basic test (code below) but Ixnetwork.Traffic.TrafficItem.Generate is not working correctly. But I get the error when I try ixnetwork.traffic.apply() (pic below) .   

```
def doBasicTest():
  IP = '22.254.11.181'
  test_platform = TestPlatform( IP , platform='linux', log_file_name='restpy.log')
  sess = test_platform.Sessions.find(Id=8020)
  ix = sess.Ixnetwork

  ix.LoadConfig(Files('ixNetworkTestSample.ixncfg'))

  traffItem = ix.Traffic.TrafficItem.find()
  tGenerated = traffItem[0].Generate()
  ix.Traffic.Apply()
```

![image](https://user-images.githubusercontent.com/18043765/51283519-ae88a600-19a5-11e9-92ed-fffafff2db4f.png)

￼
 I believe the problem is on RestPy's end b/c if I comment out traffItem[0].Generate() and comment out ix.Traffic.Apply() and then use the API browser to run both those methods, ie run a generate() method on the trafficItem and run the Apply method on Traffic, then it will work with no problems. (At /api/v1/sessions/8020/ixnetwork/traffic/trafficItem/2 via clicking Operations then choosing the Generate method and clicking commit, and same-ish thing for traffic.apply()). 

Also, if I go into the directory of my python library to [pythonpath]/ixnetwork_restpy/connection.py, and add a sleep command (shown below, line 173) then it works every time. 

￼
![screen shot 2019-01-16 at 3 56 45 pm](https://user-images.githubusercontent.com/18043765/51284012-420ea680-19a7-11e9-97dd-cebd2b91e884.png)

**Comment by hubertgee** [comment]
Hi Brodski,

Please open a support case by sending an email to support.ix@keysight.com<mailto:support.ix@keysight.com>.

By the looks of your code, since you are connecting to platform=’linux’, your session ID should not be 8020.  Linux session ID begins with 1.  The session ID 8020 looks like a Windows Connection Mgr session ID.  If you are connecting to an existing opened session, please take a look at the below ample script link:

https://github.com/OpenIxia/IxNetwork/blob/master/RestPy/SampleScripts/connectToExistingConfig.py

For basic test, follow this sample script. It has everything you need …
https://github.com/OpenIxia/IxNetwork/blob/master/RestPy/SampleScripts/bgpNgpf.py

Also, get the latest ixnetwork_restpy by pip.

Regards,
Hubert Gee

**Comment by Brodski** [comment]
Hello,

I tried making some changes to my code and looked deeper into the problem but I do not think I am doing anything wrong. The platform is linux, if I put 'windows' then it raises ConnectionError (which the doc describes under TestPlatform.__init__) occurs when "attempting to connect to a linux host with platform=`windows`" 

I do not think my code is wrong. I looked at the sample pieces and my code essential does the same. If I had illogical code then my code should break whether or not I have the line "time.sleep(5)" written. I could run the code up until  ix.LoadConfig(Files('ixNetworkTestSample.ixncfg')) then use Ixia's API Browser to run generate for the trafficItem and it will work (api/v1/sessions/8020/ixnetwork/traffic/trafficItem/operations/generate).  

The only slightly new 'discovery' is that if I sleep for 5 seconds after loading the config file (instead of in the connection.py file), then it will work. 
'''
def doBasicTest():
  IP = '22.254.11.181'
  test_platform = TestPlatform( IP , platform='linux', log_file_name='restpy.log')
  sess = test_platform.Sessions.find(Id=8020)
  ix = sess.Ixnetwork

  ix.LoadConfig(Files('ixNetworkTestSample.ixncfg'))
  time.sleep(5)
  traffItem = ix.Traffic.TrafficItem.find()
  tGenerated = traffItem[0].Generate()
  ix.Traffic.Apply()
'''
